### PR TITLE
Remove the word 'lucene' from runLuceneQuery and buildLuceneQuery.

### PR DESCRIPTION
**Pull request by samwho**
## Description

These functions never actually used Lucene, never had anything Lucene-specific about them. They were just named this way because our internal table search used Lucene under the hood. This is changing with SQS, so we're renaming these functions to remove confusion.

### CODE REVIEW

**Comment by melohagan** [comment]
Should you also rename `LuceneUtils` as well?

**Comment by samwho** [comment]
@melohagan done! 🙂

File: packages/builder/src/components/backend/DataTable/modals/ExportModal.svelte [hunk 1]
```diff
@@ -49,7 +49,7 @@
     exportFormat = Array.isArray(options) ? options[0]?.key : []
   }
 
-  $: luceneFilter = LuceneUtils.buildLuceneQuery(appliedFilters)
+  $: luceneFilter = QueryUtils.buildQuery(appliedFilters)
```
Comment (melohagan) [inline packages/builder/src/components/backend/DataTable/modals/ExportModal.svelte:52]: NAB: rename to `query`

File: packages/frontend-core/src/fetch/DataFetch.js [hunk 2]
```diff
@@ -1,10 +1,9 @@
 import { writable, derived, get } from "svelte/store"
 import { cloneDeep } from "lodash/fp"
-import { LuceneUtils } from "../utils"
+import { QueryUtils } from "../utils"
 import { convertJSONSchemaToTableSchema } from "../utils/json"
 
-const { buildLuceneQuery, luceneLimit, runLuceneQuery, luceneSort } =
-  LuceneUtils
+const { buildQuery, luceneLimit, runQuery, luceneSort } = QueryUtils
```
Comment (melohagan) [inline packages/frontend-core/src/fetch/DataFetch.js:6]: NAB: `luceneLimit` and `luceneSort` are these still relevant?

**Review by melohagan — APPROVED** [review]
LGTM!

File: packages/account-portal
Comment (mike12345567) [inline packages/account-portal:1]: Why did the account portal sub-module reference need updated?

Comment (samwho) [reply packages/account-portal:1]: Hmm, this wasn't intentional. Not sure how it happened 🤔 Will revert.

**Review by mike12345567 — APPROVED** [review]
Very nice - good to get this sorted!

**Pull request merged**